Directory.Build.props

```diff
@@ -1,24 +1,31 @@
 <Project>
   <PropertyGroup>
-    <TreatWarningsAsErrors>False</TreatWarningsAsErrors>
 
     <LangVersion>7.2</LangVersion>
     <!-- This is strong naming, not signing-->
     <SignAssembly>true</SignAssembly>
     <!-- The MSAL.snk has both private and public keys -->
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)/build/MSAL.snk</AssemblyOriginatorKeyFile>
     <DelaySign>false</DelaySign>
-    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)/build/SolutionWideAnalyzerConfig.ruleset</CodeAnalysisRuleSet>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
This is pretty heavy. We have a bunch of Xamarin warnings that are hard to ignore. Do you manage to get a clean build?

Reply:
No. There are few errors here that are not generated by analyzers:
![image](https://user-images.githubusercontent.com/61847233/108916010-8509c200-75e2-11eb-8c4f-c80805d1abc7.png)

Reply:
You can do:   `<WarningsAsErrors>CS4014</WarningsAsErrors>` instead

Reply:
Ideally we should not have to do this as we are getting the SDL recommended rules as errors without setting this as true. But the security bugs that we have been getting are the ones appearing in the warnings. So either we keep on adding the warnings that we are getting as security bugs as and when they come or we can have this flag as true.
We can also ask the TSA team generating the security bugs, why we are getting non recommended SDL rules as bugs as they are supposed to be warnings according to the analyzers too.